mlflow/keras.py

```diff
@@ -133,4 +165,9 @@ def load_model(path, run_id=None):
     """
     if run_id is not None:
         path = mlflow.tracking.utils._get_model_log_dir(model_name=path, run_id=run_id)
-    return _load_model(os.path.join(path, "model.h5"))
+    path = os.path.abspath(path)
+    flavor_conf = _get_flavor_configuration(model_path=path, flavor_name=FLAVOR_NAME)
+    # Flavor configurations for models saved in MLflow version <= 0.7.0 may not contain a
```

Comment:
Supernit: I think this should be 0.8.0? Since we're adding the `data` key in this PR, which will go out in a version greater than 0.8.0